### DISCUSSION TITLE
Trying to install JCL but I got error

### DISCUSSION THREAD

**Discussion by william34561** [project-jedi/jcl#138] [General]
when I try to install using jcl\install.bat I got this error:

Include file "source\include\jedi\jedi.inc" not found.

 I added the jedi.inc manually from jedi folder but I still get same error. How to install these components ?

**Comment by william34561** [comment]
problem solved by using these files 
https://github.com/project-jedi/jedi

Why not included

**Reply by obones** [reply]
How did you get the source files? Via a Git Checkout? Via the "source zip" on this project?
In that case, it's quite expected as this is a multi repository project and what you used is not a distribution package.
We do daily packages available here: 

https://jcl.sf.net/daily/

**Reply by william34561** [reply]
yes I got it from github, jvcl also has daily build ? where I can find it ?
please advice how to remove old jvcl and jcl ?

**Reply by obones** [reply]
https://jvcl.sf.net/daily/ is the link but it's not as crucial as for the JCL because the only dependency is the JCL.

**Reply by william34561** [reply]
Thanks
